src/Mod/PartDesign/App/FeatureHole.cpp

```diff
@@ -95,30 +95,33 @@ const Hole::ThreadDescription Hole::threadDescription[][171] =
 
     /* ISO metric regular */
     {
-        { "M1.6", 	1.60,       0.35 },
-        { "M2", 	2.00,       0.40 },
-        { "M2.5", 	2.50,       0.45 },
-        { "M3", 	3.00,       0.50 },
-        { "M3.5", 	3.50,       0.60 },
-        { "M4", 	4.00,       0.70 },
-        { "M5", 	5.00,       0.80 },
-        { "M6", 	6.00,       1.00 },
-        { "M8", 	8.00,       1.25 },
-        { "M10", 	10.00,      1.50 },
-        { "M12", 	12.00,      1.75 },
-        { "M14", 	14.00,      2.00 },
-        { "M16", 	16.00,      2.00 },
-        { "M20", 	20.00,      2.50 },
-        { "M22", 	22.00,      2.50 },
-        { "M24", 	24.00,      3.00 },
-        { "M27", 	27.00,      3.00 },
-        { "M30", 	30.00,      3.50 },
-        { "M36", 	36.00,      4.00 },
-        { "M42", 	42.00,      4.50 },
-        { "M48", 	48.00,      5.00 },
-        { "M56", 	56.00,      5.50 },
-        { "M64", 	64.00,      6.00 },
-        { "M68", 	68.00,      6.00 },
+        { "M1.6", 	1.6,       0.35 },

```

Comment:
Despites I added the missing M4.5, M7 and M39, the sizes M1.0, M1.2 (and probably M1.4) is still missing. I'll add them with the updated PR.